### PR TITLE
Quick Start V2: Congratulation screen

### PR DESCRIPTION
**Pull request by danielebogo**
Refs. #10860 

This PR adds the success screen to the tour checklist.

qs-01

qs-02

@ScoutHarris I used the no result screen as tableview background view. It works perfectly, but I added two new public methods to expose the 2 private methods I needed. This because some method like the `viewWillAppear` and `viewWillTransition` are not triggered in this case.

**To test:**
- Put return true in shouldShowQuickStartCheacklist()
- In `QuickStartTourGuide`, _customizeListTours_, remove `QuickStartSiteIconTour()` and `QuickStartNewPageTour()` in order to complete the checklist
- As soon the list is completed you should see the congratulation screen
- Expanding the completed section, should hide/show the congratulation screen

### CODE REVIEW

**Comment by danielebogo** [comment]
@SylvesterWilmott 

simulator screen shot - iphone x - 2019-01-31 at 14 05 27

simulator screen shot - iphone x - 2019-01-31 at 14 08 36

simulator screen shot - iphone x - 2019-01-31 at 14 08 40

**Comment by SylvesterWilmott** [comment]
@danielebogo Looks great!

One thing I noticed: The direction of the chevron is inverted, it should be pointing down when the group is collapsed and up when it is expanded.

**Comment by danielebogo** [comment]
Good catch @SylvesterWilmott ! Here is the fix!

simulator screen shot - iphone x - 2019-01-31 at 15 36 14

simulator screen shot - iphone x - 2019-01-31 at 15 37 30

File: WordPress/Classes/ViewRelated/Blog/QuickStartChecklistViewController.swift [hunk 1]
```diff
@@ -105,6 +131,18 @@ private extension QuickStartChecklistViewController {
...
     }
+
+    func checkForSuccessScreen(_ collapse: Bool = false) {
+        if let dataManager = dataManager,
+            !dataManager.shouldShowCompleteTasksScreen() {
+            self.tableView.backgroundView?.alpha = 0
+            return
+        }
+
+        UIView.animate(withDuration: 0.3) {
```
Comment (nheagy) [inline WordPress/Classes/ViewRelated/Blog/QuickStartChecklistViewController.swift]: Care to put the `0.3` in the `Constants` enum?

**Pull request merged**